src/test/java/org/wltea.analyzer/core/LexemeTests.java

```diff
@@ -0,0 +1,51 @@
+package org.wltea.analyzer.core;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.wltea.analyzer.core.Lexeme;
+
+public class LexemeTest {
+
+    @Test
```

Comment:
I would consider removing hashCode tests, I don't think they're particularly useful as this method is assumed to always work.